CONTRIBUTING.md

```diff
@@ -320,6 +320,6 @@ for (int i = 0; i < loop_size; i++) ...;
 
 ## Unit Tests
 
-oneMKL uses GoogleTest for functional testing.
+oneMKL uses GoogleTest for functional testing. For more information about how to build and run Unit Tests please see [oneMKL Interfaces Testing](tests/README.md)
```

Comment:
I am not aware of this README for testing... We added [a new page](https://oneapi-src.github.io/oneMKL/building_and_running_tests.html) for testing recently, and we might want to combine the information in one place. Does it make sense to list both of them for now? Or we can list the one we plan to keep.

Reply:
Good catch, I need to combine these pages.

Reply:
I've merged some information from test/README.md to the new page and left only link to new page in test/README.md and CONTRIBUTING.md.